src/Services/Spellcheckers/Aspell.php

```diff
@@ -101,14 +101,45 @@ private function run(string $text): array
 
         $output = $process->getOutput();
 
-        return array_values(array_map(function (string $line): Misspelling {
-            [$wordMetadataAsString, $suggestionsAsString] = explode(':', trim($line));
+        $lines = $this->getAspellLinesFromOutput($output);
+
+        return $this->getMisspellingsFromOutputLines($lines);
```

Comment:
We don't need the variable and can pass this directly:

```suggestion
        return $this->getMisspellingsFromOutputLines(
            $this->getAspellLinesFromOutput($output)
        );
```